src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs

```diff
@@ -3106,7 +3106,9 @@ public void NetModuleReference()
             Assert.Equal(2, comp.Assembly.Modules.Length);
         }
 
-        [Fact, WorkItem(65499, "https://github.com/dotnet/roslyn/issues/65499")]
+        [ConditionalFact(typeof(IsEnglishLocal))]
+        [WorkItem(65499, "https://github.com/dotnet/roslyn/issues/65499")]
+        [WorkItem(71236, "https://github.com/dotnet/roslyn/issues/71236")]
```

Comment:
The 71236 issue is filed to track the product issue around how the `MetadataImageReference.DisplayPath` is used in diagnostics.